rosdep/base.yaml

```diff
@@ -560,6 +560,10 @@ curl:
   opensuse: [libcurl-devel, curl]
   rhel: [libcurl-devel, curl]
   ubuntu: [libcurl4-openssl-dev, curl]
+curlpp:
```

Comment:
Please change this to be libcurlpp-dev instead of just curlpp since we may want to have a future rule for just the runtime with the name curlpp. Our default is to copy the name of the debian package.